packages/bigquery/benchmark/bench.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
I think we'll want to update the year on this

Reply:
I keep missing this. Done.